src/pages/components/tabs/usage.mdx

```diff
@@ -161,18 +174,18 @@ are always at least two tabs and one is selected by default. Icons are optional.
 
 #### 1. Line tabs
 
-A. Label <br /> B. Indicator <br />
+A. Label <br /> B. Indicator <br /> C. Scroll button
```

Comment:
I haven't updated this quite yet—i was starting to label the indicator selected indicator and i went back to just indicator because i wondering if we would need to call out the unselected indicator and crowd the image. i could update if it's best to call out both.